docusaurus/docs/core-concepts/config-store/overview.md

```diff
@@ -1,35 +1,56 @@
-# Configuration
-Ziti has the capability to define a configuration schema for an application and provide service configuration for those applications which need additional metadata. 
```

Comment:
I do think it would be useful to keep at least the spirit of this sentence here in some form.

> ziti services can optionally have associated configurations....

Reply:
> Ziti Service Configurations, henceforth "Configs", may be associated with a service to provide metadata for the application that is being delivered by that service.